Lib/test/test_embed.py

```diff
@@ -1780,8 +1781,13 @@ def test_initconfig_api(self):
             'perf_profiling': 2,
         }
         config_dev_mode(preconfig, config)
+        using_jit = any(x.startswith("JIT") for x in get_build_info())
```

Comment:
Unfortunately this won't work for `--enable-experimental-jit=interpreter` since it doesn't raise a warning about perf profiling support (because there's no conflict with tier 2 and perf profiler)

Reply:
...and this probably does not work for Windows